src/characters.c

```diff
@@ -85,6 +85,9 @@ static bool read_vowel_rules(char *const lang) {
             break;
         }
     }
+    if ((strcmp(language, "Latin") == 0 || strcmp(language, "latin") == 0 || strcmp(language, "la") == 0 || strcmp(language, "lat") == 0) && status == RFPS_NOT_FOUND) {
+        gregorio_messagef("read_rules", VERBOSITY_INFO, 0, "Falling back on internal Latin");
```

Comment:
Perhaps the message should be "Falling back on internal Latin vowel rules".

Reply:
Done.

I also modified a message a few lines later to make the two messages more consistent.